user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Adding_time_scoped_related_parameters_to_a_trend_graph.md

```diff
@@ -0,0 +1,35 @@
+---
+uid: Adding_time_scoped_related_parameters_to_a_trend_graph
+---
+
+# Adding time scoped related parameters to a trend graph
+
+To take your insights into your data to the next level, from DataMiner 10.3.8/10.4.0 onwards, you can use the light bulb in the selection of a section of a trend graph.
```

Comment:
Which release note introduces this feature? Could you add this in the title of this pull request, and also add it in comment in the text?
For example:
```suggestion
To take your insights into your data to the next level, from DataMiner 10.3.8/10.4.0 onwards<!-- RN 12345 -->, you can use the light bulb in the selection of a section of a trend graph.
```

Reply:
done